packages/sitecore-jss-tracking/src/dataModels.ts

```diff
@@ -33,3 +33,19 @@ export interface PageViewInstance {
   /** URL of page to track - should be absolute route path but can accept any value */
   url: string;
 }
+
+export interface PageViewData {
```

Comment:
The new track endpoint uses `itemId` property instead of `pageId` one. And it is optional.
`url` is the only required parameter in the `PageViewInstance` model.